fluentd/configs.d/openshift/output-applications.conf

```diff
@@ -0,0 +1,5 @@
+<match **>
+   @type copy
+   @include output-es-config.conf
+   @include ../dynamic/es-copy-config.conf
```

Comment:
That seems reasonable.

On Wed, Jul 13, 2016 at 10:50 AM, Richard Megginson <
notifications@github.com> wrote:

> In fluentd/configs.d/openshift/output-applications.conf
> <https://github.com/openshift/origin-aggregated-logging/pull/195#discussion_r70640244>
> :
>
> > @@ -0,0 +1,5 @@
> > +<match **>
> > +   @type copy
> > +   @include output-es-config.conf
> > +   @include ../dynamic/es-copy-config.conf
>
> @ewolinetz <https://github.com/ewolinetz> ok - then I think this commit
> should include the ability to provide output-extra-* and
> output-extra-ops-*, so that customers can use the plugins we already
> include, and we should implement support for out_secure_forward in a
> separate commit
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/195/files/c79a4f8ff336fad4187dcb823b73638724df6f84#r70640244>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABz-tj6Z3q3Ow5Zia42MWLsnzBFJvrNks5qVPtEgaJpZM4JHWV8>
> .
>